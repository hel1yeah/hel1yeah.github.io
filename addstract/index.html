<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Addstract</title>
  <link rel="shortcut icon" href="images/favicone.svg" type="image/x-icon">
  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700&display=swap" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Abhaya+Libre:700&display=swap" rel="stylesheet">
  <link rel="stylesheet" href="css/animate.min.css">
  <link rel="stylesheet" href="css/style.min.css">
</head>

<body>
  <!-- header -->
  <section class="header " id="header">
    <div class="container">
      <div class="nav-wrapper wow animate__animated animate__fadeInDown  animate__delay-1s ">
        <div class="logo__wrapper">
          <img class="logo" src="images/logo.svg" alt="Logo">
        </div>
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__list-item"> <a href="#about">About us</a>
            </li>
            <li class="nav__list-item"> <a href="#advantages"> Advantages</a>
            </li>
            <li class="nav__list-item"> <a href="#preview">Preview</a>
            </li>
            <li class="nav__list-item"> <a href="#video">Video</a>
            </li>
            <li class="nav__list-item"> <a href="#contact">Contact us</a>
            </li>
          </ul>
        </nav>
        <div class="header__burger">
          <span></span>
        </div>
      </div>
      <div class="title">
        <div class="title__text wow animate__animated animate__fadeInLeft animate__delay-2s">
          <div class="title__text-top">
            <h1>Do you want to
              <span>impress</span> your clients?
            </h1>
          </div>

          <div class="button">
            <button>
              Get early access
            </button>
          </div>
        </div>
        <div class="title__images wow animate__animated animate__fadeInRight  animate__delay-2s">
          <img src="images/notepad.png" alt="notepad">
        </div>
      </div>
      <div class="slider-logo wow animate__animated animate__fadeIn animate__delay-3s">
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/behance-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/dribbble-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/facebook-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/google-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/slack-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/viber-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/behance-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/dribbble-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/facebook-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/google-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/slack-logo.svg" alt="">
          </div>
        </div>
        <div class="slider-item__wrap">
          <div class="slider-item">
            <img src="images/logo/viber-logo.svg" alt="">
          </div>
        </div>

      </div>
      <div class="progress wow animate__animated animate__fadeInUp wow animate__delay-3s" role="progressbar" aria-valuemin="0" aria-valuemax="100">
      </div>
    </div>
    <div class="second wow animate__animated animate__fadeInUp animate__delay-3s">
      Trusted by the world’s best
    </div>
  </section>
  <!-- /header -->
  <!-- panel -->
  <main class="main">
    <section class="panel">
      <div class="container">
        <div class="panel-webpage " id="about">
          <div class="panel-text">
            <div class="panel-text__top">
              The simplest way to have a professioanal webpage.
            </div>
            <div class="panel-text__descr">
              Get all the benefits of an in-wall or in-ceiling speaker system without any of the hassle. Just replace a
              few
              lights
              around your house, and start streaming crisp, clear audio, everywhere.
            </div>
          </div>
          <div class="panel-mosaic">
            <div class="panel-mosaic__item mosaic-item mosaic-item__first">
              <div class="mosaic-item-wrapper item-first">
                <img src="images/panel-item/shape.svg" alt="shape">
                <div class="mosaic-text-big amazing-color">
                  Amazing design
                </div>
              </div>
            </div>
            <div class="panel-mosaic__item mosaic-item mosaic-item__two">
              <div class="mosaic-item-wrapper item-lit">
                <img src="images/panel-item/b.svg" alt="shape">
                <div class="mosaic-text-lit free-font-color">
                  Free font
                </div>
              </div>
            </div>
            <div class="panel-mosaic__item mosaic-item mosaic-item__three">
              <div class="mosaic-item-wrapper">
                <img src="images/panel-item/teka.svg" alt="shape">
                <div class="mosaic-text-big organized-color">
                  Well organized
                </div>
              </div>
            </div>
            <div class="panel-mosaic__item mosaic-item mosaic-item__four">
              <div class="mosaic-item-wrapper item-lit">
                <img src="images/panel-item/combined.svg" alt="shape">
                <div class="mosaic-text-lit themes-color">
                  Two Themes
                </div>
              </div>
            </div>
          </div>
        </div>
        <div class="panel-display" id="advantages">
          <div class="panel-mosaic-display">
            <img src="images/devices.png" alt="devices">
          </div>
          <div class="panel-text panel-text-display">
            <div class="panel-text__top">
              Display your product
              as never before
            </div>
            <div class="panel-text__descr">
              Get all the benefits of an in-wall or in-ceiling speaker system without any of the hassle. Just replace a
              few lights
              around your house, and start streaming crisp, clear audio, everywhere.
            </div>
          </div>
        </div>
        <div class="panel-card" id="preview">
          <div class="panel-text ">
            <div class="panel-text__top">
              The simplest way to have a professioanal webpage.
            </div>
            <div class="panel-text__descr">
              Get all the benefits of an in-wall or in-ceiling speaker system without any of the hassle. Just replace a
              few lights
              around your house, and start streaming crisp, clear audio, everywhere.
            </div>
          </div>
          <div class="card">
            <div class="card-top">
              <div class="card-top__info">
                <div class="card-top__info-price">
                  $99
                </div>
                <div class="card-top__info-text">
                  Abstract v.01 is here!
                </div>
              </div>
              <div class="card-top__logo">
                <img src="images/icon.svg" alt="">
              </div>
            </div>
            <div class="card-line">
              Select goodies
            </div>
            <div class="color-tems__wrapper">
              <div class="color-tems-hover">
                <div class="color-tems__item color-tems__item-one">
                  <div class="color-tems__item-text">
                    Free font
                  </div>
                </div>
              </div>
              <div class="color-tems-hover">
                <div class="color-tems__item color-tems__item-two">
                  <div class="color-tems__item-text">
                    Amazing design
                  </div>
                </div>
              </div>
              <div class="color-tems-hover">
                <div class="color-tems__item color-tems__item-three">
                  <div class="color-tems__item-text">
                    Black & White Themes
                  </div>
                </div>
              </div>
              <div class="color-tems-hover">
                <div class="color-tems__item color-tems__item-for">
                  <div class="color-tems__item-text">
                    Organized
                  </div>
                </div>
              </div>
            </div>
            <div class="card-after__one">
              <img src="images/card-after-one.png" alt="card">
            </div>
            <div class="card-after__twoo">
              <img src="images/card-after-two.png" alt="card">
            </div>
          </div>

        </div>
        <div class="panel-video" id="video">
          <div class="panel-video__text">
            <div class="panel-video__text-top">
              <h2>Layered Video Player</h2>
            </div>
            <div class="panel-video__text-descr">
              <p>Video player like never before</p>
            </div>
          </div>
          <div class="video-gallery">
            <div class="video-gallery__play">
              <img class='gallery__play' src="images/play.svg" alt="play">
            </div>
            <div class="video-gallery__card">
              <div class="top-card">
                <img class="top-card__settings" src="images/card/settings.svg" alt="settings">
                <img class="top-card__bitmap" src="images/card/bitmap.png" alt="bitmap">
                <img class="top-card__close" src="images/card/close.svg" alt="close">
              </div>
              <div class="contact-card">
                <p>Simone Miro</p>
                <p><a href="mailto:hel1_yeah@ukr.net">simonem@gmail.com</a></p>
              </div>
              <div class="work-card">
                <div class="work-card__form">
                  <div class="date-visit">06/07</div>
                  <div class="date-descr">Last Visit</div>
                </div>
                <div class="work-card__form">
                  <div class="platform-img">
                    <p>Bё</p>
                  </div>
                  <div class="platform-descr date-descr">Platform</div>
                </div>
              </div>
              <div class="line-card">
              </div>
              <div class="description-card">
                <p>
                  Thanks for using my<span> free</span> mockup.
                  Dont forgetr to <span> appreciate</span> it and to
                  leave a <span> comment .</span>
                </p>
              </div>
            </div>
            <div class="video-gallery__img">

            </div>
          </div>
          <div class="button">
            <button>
              Get early access
            </button>
          </div>
        </div>
        <div class="panel-statistics">
          <div class="statistics-text">
            <div class="statistics-text__top">
              <h2>Statistics & Information</h2>
            </div>
            <div class="statistics-text__descr">
              <p>Numbers speak louder than words</p>
            </div>
          </div>
          <div class="statistics-information">
            <div class="statistics-information__item pie-chart">

              <!-- для создания круговой аниммированной диаграммы создаём svg рисунок а в нём круг
                значения :
                width='125'ширена 
                height='125' высота
                stroke='#3B5873' цвет обводки
                stroke-width='6px' ширина обводки
                stroke-linecap="round" закруглённость обводки
                fill='transparent' внутренний цвет 
                cx='60' ширина круга
                cy='60' высота круга
                r='54' радиус круга щитаем за формулой radius = (width / 2) - (strokeWidth * 2 )
          -->

              <svg class="pie-chart__wrap" width='140' height='140'>
                <circle class="pie-chart__circle" fill='transparent' stroke='#3B5873' stroke-width='6px'
                  stroke-linecap="round" cx='70' cy='70' r='64' />
                <text x="40" y="90" fill='#7887A7' class="pie-chart__percent">67</text>
                <text x="100" y="49" fill='#7887A7' class="pie-chart__sign">%</text>
              </svg>
              <div class="pie-chart__text">
                More visitors
                during last
                ineteract
              </div>
              <div class="pie-chart__days">
                <div class="pie-chart__days__item pie-chart__days__today">Today</div>
                <div class="pie-chart__days__item  pie-chart__days__this-week">
                  <nobr>This Week</nobr>
                </div>
                <div class="pie-chart__days__item  pie-chart__days__this-month">This Month</div>
              </div>
            </div>
            <div class="statistics-information__item interest">
              <div class="interest__descr">
                43% more users after update
              </div>
              <div class="interest__interest">
                43 <span>%</span>
              </div>
            </div>
            <div class="statistics-information__item line-chart">
            </div>
          </div>
          <div class="button__empty__wrap">
            <div class="button__empty">
              <button>
                View all data
              </button>
            </div>
          </div>


        </div>
        <div class="panel-communication">
          <div class="panel-communication__descr">
            Build your own future.
            <br>
            Download now free and get 100% boost!
          </div>
          <div class="panel-communication__buttons">
            <div class="button">
              <button>Get early access</button>
            </div>
            <div class="button__empty">
              <button>Newsletter</button>
            </div>
          </div>
        </div>
        <footer class="footer" id="contact">
          <div class="switch-language">
            English
          </div>
          <div class="nav-menu">
            <ul class="nav-menu__list">
              <li><a href="#header">Home</a></li>
              <li><a href="#about">
                  <nobr>About us</nobr>
                </a></li>
              <li><a href="#advantages">Advantages</a></li>
              <li><a href="#preview">Preview</a></li>
              <li><a href="#video">Video</a></li>
            </ul>
            <ul class="nav-menu__support">
              <li><a href="#">Terms & Conditions</a></li>
              <li><a href="#">Privacy policy</a></li>
              <li><a href="#">Support</a></li>
              <li><a href="#">Chrom Extension</a></li>
            </ul>
          </div>
          <div class="contacts">
            <div class="contacts__address">
              <a target="_blank" href="https://goo.gl/maps/hJZSi8gbAgYWHiCy6"> 231 North Star 34th Street, Suite #100
                <br>
                Cupertino, 98103
              </a>
            </div>
            <div class="contacts__icons">
              <div class="contacts__icons__item">
                <a href="https://dribbble.com/" target="_blank">
                  <img src="images/icons/icons-dribbble.svg" alt="icons" width='33px' height='33px'>
                </a>
              </div>
              <div class="contacts__icons__item"><a href="#"><img src="images/icons/icons-Instagram.svg" alt="icons"
                    width='33px' height='33px'></a></div>
              <div class="contacts__icons__item"><a href="#"><img src="images/icons/icons-behance.svg" alt="icons"
                    width='33px' height='33px'></a></div>
              <div class="contacts__icons__item"><a href="#"><img src="images/icons/icons-facebook.svg" alt="icons"
                    width='33px' height='33px'></a></div>
              <div class="contacts__icons__item"><a href="#"><img src="images/icons/icons-medium.svg" alt="icons"
                    width='33px' height='33px'></a></div>
            </div>
          </div>
          
        </footer>
      </div>
      <div class="copyrigts">
        <div class="container">
          <div class="copyrigts__wrapper">
            <div class="copyrigts__logo"><a href="#"><img src="images/logo.svg" alt=""></a>
            </div>
            <div class="copyrigts__team">
              <a href="#"> © 2017 Copyrigts are Creative Commons</a>

            </div>
            <div class="copyrigts__thanks">Don’t forget to give credit 💙</div>
          </div>
        </div>
      </div>
    </section>
  </main>





  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="js/libs.min.js"></script>
  <script src="js/wow.min.js"></script>
  <script src="js/main.js"></script>
</body>

</html>