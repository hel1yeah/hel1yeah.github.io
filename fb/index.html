<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <link rel="icon" href="./favicon.ico" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Vite App</title>
    <script type="module" crossorigin src="./assets/index.3f049a3c.js"></script>
    <link rel="modulepreload" href="./assets/vendor.4a3f02d2.js">
    <link rel="stylesheet" href="./assets/index.eba88987.css">
  </head>
  <body>
    <div id="app"></div>
    
  </body>
</html>
