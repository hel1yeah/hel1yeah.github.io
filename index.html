<!DOCTYPE html>
<html lang="ru">

<head>
  <meta charset="UTF-8">
  <title>MyPortfolio</title>
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta name="description"
    content="My portfolio, which I’ve been modifying forever. Here I learn to add all sorts of chips ideally.">
  <meta name="keywords"
    content="portfolio, first site, studying, developing, looking for work, looking for friends by interests, looking for knowledge, growth">
  <meta name="robots" content="index, follow">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta name="language" content="Russian">
  <meta name="author" content="Yura Larsen">
  <link rel="stylesheet" type="text/css" href="css/style.css">
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
    integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
  <link rel="shortcut icon" href="img/triangle.png" type="image/x-icon">
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-149779182-1"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-149779182-1');
  </script>
</head>
<body>
  <!--=============   home page	 =============-->
  <header class="header">
    <div class="container">
      <div class="wrapper_header">
        <nav class="wrapper_header-nav">
          <div class="type_logo ">
            <p><span>My</span>Portfolio</p>
          </div>
          <div class="menu">
            <ul>
              <li><a href="#about_me">Обо мне</a></li>
              <li><a href="#skills">Навыки</a></li>
              <li><a href="#work">Работы</a></li>
            </ul>
          </div>
          <div class="connection">
            <p>hel1<span id="blink">_</span>yeah@ukr.net</p>
          </div>
        <div class="wrapper-menu">
          <div class="line-menu half start"></div>
          <div class="line-menu"></div>
          <div class="line-menu half end"></div>
        </div>
        </nav>
        <div class="top_text">
          <div class="bleach">
            <h1>Дело, которым я живу</h1>
          </div>
          <p>вёрстка & frontend</p>
        </div>
        <div class="arrow">
          <a href="#about_me"><i class="fas fa-long-arrow-alt-down"></i></a>
        </div>
      </div>


      <div class="arrow_up-wrapper ">
        <div class="arrow">
          <a href="#"><i class="fas fa-long-arrow-alt-up"></i></a>
        </div>
      </div>

    </div>
  </header>
  <!--=============   /home page	 =============-->
  <!--=============   about me	 =============-->
  <div id="about_me" ></div>
  <div class="wrapper_skills social_border">
    <div class="container">
      <div class="title_skills">
        <h2>Обо мне</h2>
      </div>
      <div class="box_skills_wrapper">
        <div class="left_box_skills">
          <div class="box_skills">
            <h3>Бэкграунд</h3>
            <p>Junior Front - End Developer. 
              В мир Web-разработки попал случайно, по работе нужно было разобраться в некоторых тонкостях этой сферы.
              И я разобрался.
              Потратив огромное количество часов на всевозможные статьи и видео, эта вселенная оставила очень большое впечатление.
            </p>
            <p>Настолько большое, что вместо того чтобы отдыхать после работы, я, сам того не замечая, засиживался до 2 часов ночи за вёрсткой.
              Так Front-End стал для меня хобби, которым мне хочется  заниматься. 
            </p>
            <p>Изучив самостоятельно HTML5, CSS3, Photoshop, мне всё ещё казалось, что я чего - то не знаю.
              Следующий шаг - курсы, к сожалению, выделил из них мало чего полезного для себя, потому что большую часть материала знал до этого.
              Дальше - я взялся за изучение: jQuery, Gulp, GitHub, Git, Figma. Сейчас я изучаю JavaScript, дальше будет React.
            </p>
             <p>
             </p>
            <p>Если у Вас есть советы, пожелания, предложения по практике или работе, буду рад знакомству!
             </p>
          </div>
        </div>
        <div class="right_box_skills">
          <div class="title_skills">
            <h3>Заслуги</h3>
          </div>
          <div class="square_container">
            <div class="square_items">
              <div class="square_outside">
                <div class="square_inside">
                  <p>101%</p>
                </div>
              </div>
              <div class="square_signature">
                <p>Обучаемость</p>
              </div>
            </div>
            <div class="square_items">
              <div class="square_outside">
                <div class="square_inside">
                  <p>777</p>
                </div>
              </div>
              <div class="square_signature">
                <p>Удача</p>
              </div>
            </div>
            <div class="square_items">
              <div class="square_outside">
                <div class="square_inside">
                  <p>100%</p>
                </div>
              </div>
              <div class="square_signature">
                <p>Преданность <br> Королю</p>
              </div>
            </div>
          </div>
          <div class="wrapper_social">
            <div class="title_skills">
              <h3>Связь</h3>
            </div>
            <div class="social" id="social_border">
              <ul>
                <li>
                  <a href="https://www.facebook.com/YuraLarsen" title="facebook" target="_blank">facebook.com/YuraLarsen
                  </a>
                  <span class="fb">
                    <i class="fab fa-facebook-square"></i>
                  </span>
                </li>
                <li>
                  <a href="https://www.linkedin.com/in/yura-larsen/" title="linkedin"
                    target="_blank">linkedin.com/in/yura-larsen
                  </a>
                  <span class="li fb">
                    <i class="fab fa-linkedin"></i>
                  </span>
                </li>
                <li>
                  <a href="https://t.me/hel1_yeah" title="telegram" target="_blank">telegram: @hel1_yeah
                  </a>
                  <span class="tel fb">
                    <i class="fab fa-telegram"></i>
                  </span>
                </li>
                <li>
                  <a href="mailto:hel1_yeah@ukr.net" title="ukr.net" target="_blank">hel1_yeah@ukr.net
                  </a>
                </li>
                <li>
                  <a href="https://github.com/hel1yeah" title="github" target="_blank">github: hel1yeah</a>
                  <span class="git fb"><i class="fab fa-github"></i></span>
                </li>
                <li>
                  <a href="mailto:hel1rockua@gmail.com" title="google" target="_blank">hel1rockua@gmail.com</a>
                  <span class="google fb"><i class="fab fa-google"></i></i></span>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <!--=============   /about me  =============-->
  <!--=============   skills	 =============-->
  <div id="skills"></div>
  <div class="container">
    <div class="experience-wrapper">
      <div class="title_skills">
        <h2>Навыки</h2>
      </div>
      <div class="experience">
        <div class="experience_items">
          <div class="disk_experience">
            <div class="inner_circle">
              <div class="inner_circle_text">
                <p>83%</p>
              </div>
            </div>
          </div>
          <div class="items_text">
            <p>Photoshop, Figma</p>
          </div>
        </div>
        <div class="experience_items">
          <div class="disk_experience">
            <div class="inner_circle">
              <div class="inner_circle_text">
                <p>91%</p>
              </div>
            </div>
          </div>
          <div class="items_text">
            <p>html</p>
          </div>
        </div>
        <div class="experience_items">
          <div class="disk_experience">
            <div class="inner_circle">
              <div class="inner_circle_text">
                <p>85%</p>
              </div>
            </div>
          </div>
          <div class="items_text">
            <p>css3</p>
          </div>
        </div>
        <div class="experience_items">
          <div class="disk_experience">
            <div class="inner_circle">
              <div class="inner_circle_text">
                <p>33%</p>
              </div>
            </div>
          </div>
          <div class="items_text">
            <p>Git, GitHub</p>
          </div>
        </div>
        <div class="experience_items">
          <div class="disk_experience">
            <div class="inner_circle">
              <div class="inner_circle_text">
                <p>0.1%</p>
              </div>
            </div>
          </div>
          <div class="items_text">
            <p>Лень</p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <!--=============   /skills	 =============-->
  <!--=============   /work	 =============-->
  <div id="work"></div>
  <div class="particles_wrapper">
  <div id="particles"></div>
  <div class="container">
    <div class="wrapper_gallery">
      <div id="particles"></div>
      <div class="title_skills working ">

        <h2>Работы</h2>
      </div>
      <div class="gallery_working">
        <div class="gallery_img prechu_item">
          <img src="img/1.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>Prechu</h5>
              <p>Первые попытки верстки <br>
                <a href="download/Prechu.jpg" target="_blank" download>Хочу скачать макет.</a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img tell-net_item">
          <img src="img/2.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>Tell-Net</h5>
              <p>Здесь я сам решил быть дизайнером и верстальщиком,но что-то пошло не так (:</p>
            </div>
          </div>
        </div>
        <div class="gallery_img golden-net_item">
          <img src="img/3.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>Golden</h5>
              <p>Верстка вместе с online webinar
                <br>
                <a href="download/Golden.jpg" target="_blank" download>Хочу скачать макет.</a>
                <br>
                <a target="_blank" href="https://hel1yeah.github.io/golden/">Хочу посмотреть сайт</a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img fdisign_item">
          <img src="img/4.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>Fdisign</h5>
              <p>Одно из заданий онлайн вебинаров
                <br>
                <a href="download/fdesign.jpg" target="_blank" download>Хочу скачать макет.</a></p>
            </div>
          </div>
        </div>
        <div class="gallery_img benelli_item">
          <img src="img/5.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>Benelli</h5>
              <p>очень умный текст
                <br>
                <a href="download/Benelli.jpg" download target="_blank">Хочу скачать макет.</a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img tajam_item">
          <img src="img/6.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>Tajam</h5>
              <p>Одна из последних и самых больших робот
                <br>
                <a href="download/tajemr.jpg" target="_blank" download>Хочу скачать макет.
                </a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img misocial_item">
          <img src="img/7.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img">
              <h5>MISOCIAL</h5>
              <p>Теперь уже и в адаптиве, как говорится)
                <br>
                <a href="download/misocial.jpg" target="_blank" download>Хочу скачать макет.
                </a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img surf_item">
          <img src="img/8.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img surf_item-img">
              <h5>Surf</h5>
              <p>Сверх крутой макет, сделаный с помощью марафона по вёрстке с Вадимом Прокопчуком
                <br>
                <a href="download/go_surf.jpg" target="_blank" download>Хочу скачать макет.
                </a>
                <br>
                <a target="_blank" href="https://hel1yeah.github.io/surf">Хочу посмотреть сайт</a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img astyle_item">
          <img src="img/10.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img astyle-img">
              <h5>A-style</h5>
              <p>Реально работающий проєкт. Оперативная полиграфия и широкоформатная печать любого размера
                <br>
                <a href="download/a-style.jpg" target="_blank" download>Хочу скачать макет.
                </a>
                <br>
                <a target="_blank" href="https://a-style.000webhostapp.com/">Хочу посмотреть сайт</a>
              </p>
            </div>
          </div>
        </div>
        <div class="gallery_img cantral103_item">
          <img src="img/9.jpg" alt="alt">
          <div class="wrapper_img_text">
            <div class="text_img cantral103-img">
              <h5>Централь 103</h5>
              <p>Национальная ИТ-система, контролирующая качество оказания экстренной медицинской помощи и ускоряет прибытия первых реагувальников
                <br>
                <a href="download/cantral103.jpg" target="_blank" download>Хочу скачать макет.
                </a>
                <br>
                <a target="_blank" href="https://hel1yeah.github.io/central103">Хочу посмотреть сайт</a>
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  </div>
  <!--=============   /work	 =============-->
  <!--=============   footer	 =============-->
  <footer>
      <div class="container">
        <p>2019-2099 / Смотрю в будущее</p>
      </div>
  </footer>
  
  <!--=============   /footer  =============-->
  <!--=============   script  =============-->
  <script src="libs/jquery.min.js"></script>
  <script src="libs/particles/particles.min.js"></script>
  <script src="js/maine.js"></script>
  <!--=============   /script  =============-->
  
</body>

</html>
