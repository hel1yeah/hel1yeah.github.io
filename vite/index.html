<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <link rel="icon" href="/favicon.ico" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Vite App</title>
    <script type="module" crossorigin src="/assets/index.80ae4162.js"></script>
    <link rel="modulepreload" href="/assets/vendor.5b0e4a6e.js">
    <link rel="stylesheet" href="/assets/index.468ba579.css">
  </head>
  <body>
    <div id="app"></div>
    
  </body>
</html>
