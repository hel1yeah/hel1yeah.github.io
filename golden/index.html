<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Golden</title>
  <link rel="stylesheet" href="css/style.min.css">
</head>

<body>

  <body>
    <div id="upwards"></div>
    <div id="arrow">
      <a href="#upwards">
        <img src="images/arrow.png" width="50" height="50"></a>
    </div>
    <header>
      <div class="heading  ">
        <div class="heading__wrap">
          <a class="logo" href="#">
            <img src="images/logo.png" alt="Golden" class="logo">
          </a>
          <nav class="navigator">
            <ul class="menu">
              <li>
                <a href="#upwards">Home</a>
              </li>
              <li>
                <a href="#Services_WWW">Services
                </a>
              </li>
              <li>
                <a href="#Portfolio_WWW">Portfolio
                </a>
              </li>
              <li>
                <a href="#About_WWW">About
                </a>
              </li>
              <li>
                <a href="#contact_WWW">Contact
                </a>
              </li>
            </ul>
          </nav>
          <div class="header__burger">
            <span></span>
          </div>
        </div>

      </div>
      <div class="container" id="upwards">
        <div class="titles">
          <div class="titles_first">
            Welcome To Our Studio!Jesus.
          </div>
          <h1>
            It’s nice to Meet you
          </h1>
        </div>
        <a class="button" href="#">Tell Me More</a>
      </div>
    </header>
    <section id="servises">
      <div class="container" id="Services_WWW">
        <div class="title">
          <h2>
            Servises
          </h2>
          <p>
            Hava nagila hava Hava nagila hava!
          </p>
        </div>
        <div class="servises  ">
          <div class="servises_item">
            <img src="images/icon1.png" alt="Услуга">
            <h3>
              E-Commerse
            </h3>
            <p>
              "I will be meeting again with first responders, including St John’s ambulance and front-line support
              staff.
              I plan on meeting with family
            </p>
          </div>
          <div class="servises_item">
            <img src="images/icon2.png" alt="Услуга">
            <h3>
              Responsive Web
            </h3>
            <p>
              "I will be meeting again with first responders, including St John’s ambulance and front-line support
              staff.
              I plan on meeting with family
            </p>
          </div>
          <div class="servises_item">
            <img src="images/icon3.png" alt="Услуга">
            <h3>
              Web Secirity
            </h3>
            <p>
              "I will be meeting again with first responders, including St John’s ambulance and front-line support
              staff.
              I plan on meeting with family
          </div>
        </div>
      </div>
    </section>
    <section id="portfolio" class="portfolio">
      <div class="container" id="Portfolio_WWW">
        <div class="title">
          <h2>
            Our portfolio
          </h2>
          <p>
            England student teching Ukrainian language.Beach.!
          </p>
        </div>
        <div class="work">
          <img src="images/img1.jpg" alt="Работа">
          <img src="images/img2.jpg" alt="Работа">
          <img src="images/img3.jpg" alt="Работа">
        </div>
      </div>
    </section>
    <section id="About_WWW">
      <div class="container">
        <div class="title" >
          <h2>
            About Us
          </h2>
          <p>
            Front-end develiper in UA, Slava Ukraune
          </p>
        </div>
        <div class="finish_R">
          
          <div class="title_R">
            <h2>
              July 2010
              Our Humble Beginnings
            </h2>
            <p>Proin iaculis purus consequat sem cure
            digni ssim. Donec porttitora entum suscipit
            aenean rhoncus posuere odio in tincidunt. Proin
            iaculis purus consequat sem cure digni
            ssim. Donec porttitora entum suscipit.
            </p>
            </div>
            <img src="images/about1.jpg" alt="finishR">
          </div>  
          <div class="finish_L">
            <img src="images/about2.jpg" alt="finishL">
            <div class="title_L">
              <h2>
                January 2011
                Facing Startup Battles

              </h2>
              <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in
              tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.
            </p>
            </div>
            </div>
            <div class="finish_R">
              
              <div class="title_R">
                <h2>
                  December 2012
                  Enter The Dark Days

                </h2>
                <p>IT engineers are employed in many different industries to create, install, or troubleshoot technology
                  and software programs. If you would like to become an IT </p>
              </div>
              <img src="images/about3.jpg" alt="finishRR">
              </div>
              <div class="finish_L">
                <img src="images/about4.jpg" alt="finishLL">
                <div class="title_L">
                  <h2>
                    february 2014
                    Our Triumph

                  </h2>
                  <p>
                    Proin iaculis purus consequat sem cure
                  digni ssim. Donec porttitora entum suscipit
                  aenean rhoncus posuere odio in tincidunt. Proin
                  iaculis purus consequat sem cure digni
                  ssim. Donec porttitora entum suscipit aenean.
                 </p>
                </div>
              </div>
            </div>
    </section>
    <footer>
      <div class="container">
        <div class="footer" id="contact_WWW">
          <p>
            <blink>Telegram @Hel1<font color="#FF4500">_</font>Yeah. 27.03.2019
          </p>
        </div>
      </div>
      </div>
    </footer>
    </div>
  </body>

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="js/libs.min.js"></script>
  <script src="js/main.js"></script>
</body>

</html>